index.php

```diff
@@ -19,9 +28,31 @@
     <script defer src="js/index.js"></script>
 </head>
 
-<body>
-
-<h1>Website Template</h1>
 
+<body>
+    <header>
+        <h1>Games Collection</h1>
+    </header>
+    <main>
+        <span class="game">
+            <?php
+
+            $gameModel = new GameModel($db);
+            $games = $gameModel->getAllGames();
+
+            foreach ($games as $game) {
+                echo "<h2>$game->name</h2>";
+                echo "<p>Franchise: $game->franchise</p>";
+                echo "<p>Price: $game->price</p>";
+                echo "<p>Genre: $game->genre</p>";
+            }
```

Comment:
You should make a function/object (your choice) that handles displaying the games for you. It could be useful on other pages.

You should make it so the function returns the string of game data rather than echoing it - you an then write some unit tests for it! woooo